src/main/java/com/newrelic/telemetry/opentelemetry/export/DeltaLongCounter.java

```diff
@@ -0,0 +1,18 @@
+package com.newrelic.telemetry.opentelemetry.export;
+
+import io.opentelemetry.sdk.metrics.data.MetricData.LongPoint;
+
+public class DeltaLongCounter {
+
+  private LongPoint previousValue = null;
+
+  long delta(LongPoint newValue) {
+    if (previousValue == null) {
+      previousValue = newValue;
+      return previousValue.getValue();
+    }
+    long result = newValue.getValue() - previousValue.getValue();
+    previousValue = newValue;
+    return result;
+  }
+}
```

Comment:
Aside from types, this is identical to the `DeltaDoubleCounter`.  Can these be combined?

Reply:
I don't see an easy, non-crazy way to combine them. LongPoint and DoublePoint are distinct classes because adding generics would make every single value get boxed/unboxed, creating too much garbage for some people.

Reply:
I am hoping that these classes will be very short-lived, as well, since I'm working on making the aggregation/export pipeline not require us to do our own delta calculations. I think I'm ok with a couple lines of code duplication to keep things simple.

Reply:
That's fair.  I hadn't considered the boxing aspect.  Bummer, but I understand.